batch/batch/server/server.py

```diff
@@ -504,17 +595,17 @@ def close_batch(batch_id):
     return jsonify({})
 
 
-def update_job_with_pod(job, pod):
+def update_job_task_with_pod(job_task, pod):
     if pod:
         if pod.status.container_statuses:
             assert len(pod.status.container_statuses) == 1
             container_status = pod.status.container_statuses[0]
-            assert container_status.name == 'default'
+            assert container_status.name in ['input', 'default', 'output']
```

Comment:
In a follow up PR can we change the name of `'default'` to `'main'` everywhere? I think this will incur a change in client.py as well.